pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -5994,9 +6070,9 @@ def test_downsample_in_time_varying_integration_time(resample_in_time_file):
     uv_object.integration_time[inds01[-2:]] += initial_int_time
     uv_object.Ntimes = np.unique(uv_object.time_array).size
     min_integration_time = 2 * np.amin(uv_object.integration_time)
-    uvtest.checkWarnings(
-        uv_object.downsample_in_time, [min_integration_time], nwarnings=0
-    )
+    with pytest.warns(None) as record:
```

Comment:
Why do we explicitly check that no warnings are raised here? (I approve of the move to using `pytest.warns`, but just curious why we're doing this.)

Reply:
I'm not perfectly sure, but I didn't want to remove an existing check. I think maybe it was because we wanted to make sure that we were doing things consistently when we changed the `integration time` and `time_array` (a warning is issued when there's inconsistencies between those 2 arrays).

Reply:
I can take it out if you think we don't need it, or I can add a comment.

Reply:
Fair enough. Yes, I think a comment explaining we don't want to raise that warning would help.